matlab/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+find_package(Matlab REQUIRED)
+matlab_add_mex(NAME finufft_mex SRC finufft.cpp LINK_TO finufft OUTPUT_NAME finufft R2018a)
```

Comment:
No it will automatically fall back to the previous C api for complex number layout, see [doc](https://cmake.org/cmake/help/latest/module/FindMatlab.html#command:matlab_add_mex).